test/helpers/projects_helper_test.rb

```diff
@@ -30,7 +29,8 @@
     it 'returns the deployment alert data' do
       job = project.jobs.create!(command: 'cat foo', user: users(:deployer), status: 'failed')
       deploy = stage.deploys.create!(reference: 'master', job: job, project: project)
-      expected_title = "#{deploy.updated_at.strftime('%Y/%m/%d %H:%M:%S')} Last deployment failed! #{deploy.user.name} failed to deploy '#{deploy.reference}'"
+      expected_title = "#{deploy.updated_at.strftime('%Y/%m/%d %H:%M:%S')} Last deployment failed! " \
+        "#{deploy.user.name} failed to deploy '#{deploy.reference}'"
```

Comment:
That works? 2 strings next to each other with no + in there? I would personally prefer the +. It looks odd otherwise.

Reply:
Okay. I learned alot, so in the end, this is the only issue I have with the rubocop default style changes. What do you think about sticking a plus in there?

Reply:
works but rubocop disagrees :)
```
test/helpers/projects_helper_test.rb:32:102: C: Style/LineEndConcatenation: Use \ instead of + or << to concatenate those strings.
      expected_title = "#{deploy.updated_at.strftime('%Y/%m/%d %H:%M:%S')} Last deployment failed! " +
```